index.html

```diff
@@ -60,7 +63,16 @@
       </div>
     </div>
   </div>
-    
+  
+  <div>
+
+  <div >
+    <a href="instruction.html"><img src="images/surprise.jpeg" class="surprise" style="width:100px;height:100px;"></a>
```

Comment:
Let's try to use SVGs instead of jpegs and pngs. That way, they will be both transparent background and look good in all sizes. 
Let's also try to use images with a similar style, preferably without skin color, i.e. black and white. 

This should be a good source for images: https://www.flaticon.com/search?word=hand&order_by=4
For pinching, search for "ok"
For v sign, search for "peace"

Reply:
It seems only pngs for free in this website, svgs need a payment.

Reply:
Oops, I meant this one https://www.svgrepo.com/vectors/hand/